platform-coop-toolkit/pcc
# Fix events

## Description

Changes event query behavior, changes date displayed on cards and project title color

## Steps to test

1. Create a page and set the new template "Events"
2. Open the new page created
3. You should be able to see the events and not to see child events
4. Open a single event
5. You should be able to see no line above image
6. Create a project
7. The title has change the color

**Expected behavior:** Events page must not contain child events
